lib/matplotlib/path.py

```diff
@@ -493,9 +493,15 @@ def contains_point(self, point, transform=None, radius=0.0):
         """
         if transform is not None:
             transform = transform.frozen()
-        result = _path.point_in_path(point[0], point[1], radius, self,
-                                     transform)
-        return result
+        from .transforms import Affine2D
```

Comment:
Any reason to have this here and not at the top of the module as python convention require?